crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -393,3 +409,12 @@ impl ExternalApiWireError {
         ExternalApiWireError::InternalError(e.to_string())
     }
 }
+
+#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize, SerializedBytes)]
+/// Info about all the apps installed in the conductor, returned by [`AdminResponse::ListApps`]
+pub struct ListAppsResponse {
```

Comment:
I'm not sure the right way to do this is by storing active and inactive in two separate lists.  It seems like we should just have one list and then in the conductor-api libraries implement accessor method(s) to get the active ones and the inactive ones or any other sub-set if that's what we want.  I think that makes the API simpler and more future proof.

Reply:
I guess this depends on whether we want to expose the fact that the conductor state holds the two lists explicitly right now.  @maackle @artbrock?

Reply:
Yeah that was one of the questions for me, actually the first implementation I did was with `Vec<InstalledAppInfo>`. But it seemed easier send back the two lists this way since I need them separated either way... 

Happy to reverse back if you want :)

Reply:
I think we did it this way so that the user couldn't accidentally blow through all apps and ignore their active status. Basically, this way is more opinionated, making the assumption that it's much more likely that the user has to deal with active apps separately from inactive, than that they actually want the full list.

If it turns out to be common to want the full list, then I would support changing the API. Otherwise, I'm indifferent, because it's trivial to map between the two shapes in one or two LOC.

Reply:
It seems like we are leaning toward having a single data list and either add filters on the initial query or on results accessors.  I think this is a more resilient API.

Reply:
I was thinking that during the request, an enum could be given: 
```
enum AppsListStatus {
  Active
  Inactive
  Both
}
```
and one variant of this enum would be passed as the input to the request.

Reply:
Okey, I'll change the API to be one list and have a filter like @Connoropolous  suggested. Actually maybe a ```rust filter: Option<AppStatusFilter>, AppStatusFilter { Active, Inactive }``` makes more sense?